Corrected text to include Validated Content (#495)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/automation-hub/con-rh-certified-synclist.adoc

= About Red Hat Certified collections synclists

A synclist is a curated group of Red Hat Certified and validated collections that is assembled by your organization administrator that syncs to your local {HubName}. 
Use synclists to manage only the content that you want and exclude unnecessary collections.

> You can design and manage your synclist from the content available as part of Red Hat Certified and Validated Collections on console.redhat.com

Reviewer: ```suggestion
You can design and manage your synclist from content in the Red Hat Certified and Validated Collections on `console.redhat.com`.
```
